pyfolio/tears.py

```diff
@@ -86,7 +86,6 @@ def create_full_tear_sheet(returns,
                            volumes=None,
                            percentile=None,
                            turnover_denom='AGB',
-                           set_context=True,
```

Comment:
Why was this removed?

Reply:
It was a useless parameter: as far as I could tell, it was passed around to all `create_x_tear_sheet` functions, which did nothing with it. Perhaps `seaborn` makes use of it, but I don't see how it would... @twiecki can you confirm that it's safe to remove this?

Reply:
@eigenfoo it looks to me like this parameter is consumed by the `@customize` decorator in `pyfolio.plotting`:

```python
def customize(func):
    """
    Decorator to set plotting context and axes style during function call.
    """
    @wraps(func)
    def call_w_context(*args, **kwargs):
        set_context = kwargs.pop('set_context', True)
        if set_context:
            with plotting_context(), axes_style():
                return func(*args, **kwargs)
        else:
            return func(*args, **kwargs)
    return call_w_context
```

That decorator is applied to most of the `create_*_tear_sheet` methods.

In general, this strikes me as a scary pattern, because it would be very easy for us to accidentally forget to pass `set_context` somewhere, and we don't have any test coverage enforcing that we handle that parameter correctly. I think trying to fix that problem is probably outside the scope of this PR though. At the very least, I'd revert the removal of that parameter for now.

Reply:
(In point of fact, it looks like we're failing to pass set_context to several of the tear sheet calls here...)

Reply:
Reverted. We'll definitely need to revisit this in a later PR.

Reply:
Can you create an issue to track the fact that we're passing this inconsistently?